handlers/holiday-stop-processor/src/main/scala/com/gu/holidaystopprocessor/Salesforce.scala

```diff
@@ -12,16 +12,8 @@ import com.typesafe.scalalogging.LazyLogging
 
 object Salesforce extends LazyLogging {
 
-  /*
-   * Zuora takes 3-4s to add each credit to a sub so this should take ~ 10 mins,
-   * which is within the 15 min max that a lambda can run.
-   * As the lambda runs each hour, all should easily be processed within the
-   * 24 hour window available.
-   *
-   * Beware! This isn't a scalable solution.  If all the products between them
-   * have > 5000 credit requests to be processed in a 24 hour window some will be missed.
-   */
-  private val batchSize = 66
+  //TODO: Move the batching into a step function/state machine
```

Comment:
Would be nice to update the comment with some justification for the batch size, I'm not sure what to put though.. I've made something up in the suggestion
```suggestion
  // we choose 800 because we run them in parallel across all products and any that are not completed would be picked up after timeout, we don't need to ensure we complete all within the lambda timeout any more.
```